### PR TITLE
Add REST example

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[ef849d9](https://github.com/OfficeDev/outlook-dev-docs/pull/466/commits/ef849d97843d4e9d7b38ff7fec1434f237dd3fab)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[ef849d9](https://github.com/OfficeDev/outlook-dev-docs/pull/466/commits/ef849d97843d4e9d7b38ff7fec1434f237dd3fab)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[ef849d9](https://github.com/OfficeDev/outlook-dev-docs/pull/466/commits/ef849d97843d4e9d7b38ff7fec1434f237dd3fab)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/metadata-for-an-outlook-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-add-rest-example/docs/add-ins/metadata-for-an-outlook-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C20%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201812200057273223-466%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RbAHzKRjyJFwwMtW2MX7S7FjUSm8IYmYVwN8P3rBGNk%3D&st=2018-12-20T00%3A53%3A27Z&se=2019-01-20T00%3A58%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/metadata-for-an-outlook-add-in.md [hunk 1]
```diff
@@ -273,6 +273,54 @@ GET https://outlook.office.com/api/v2.0/Me/Events?$filter=SingleValueExtendedPro
 
 For other examples that use REST to get single-value MAPI-based extended properties, see [Get singleValueExtendedProperty](/graph/api/singlevaluelegacyextendedproperty-get?view=graph-rest-1.0).
 
+The following example shows how to get an item and its custom properties. In the callback function for the `done` method, `item.SingleValueExtendedProperties` will give a list of the requested custom properties.
```
Comment (kbrandl) [inline docs/add-ins/metadata-for-an-outlook-add-in.md]: "will give a list of the requested custom properties." -> "contains a list of the requested custom properties."

File: docs/add-ins/metadata-for-an-outlook-add-in.md [hunk 2]
```diff
@@ -273,6 +273,54 @@ GET https://outlook.office.com/api/v2.0/Me/Events?$filter=SingleValueExtendedPro
...
+                        console.log(JSON.stringify(error));
+                    }
+                );
+            }
+            else {
+                console.log(JSON.stringify(asyncResult));
+            }
+        }
+    );
+```
```
Comment (kbrandl) [inline docs/add-ins/metadata-for-an-outlook-add-in.md]: Should fix indentation of the last 6 lines. Should read like this:

```
Office.context.mailbox.getCallbackTokenAsync(
    {
        isRest: true
    },
    function (asyncResult) {
        if (asyncResult.status === Office.AsyncResultStatus.Succeeded
            && asyncResult.value !== "") {
            let item_rest_id = Office.context.mailbox.convertToRestId(
                Office.context.mailbox.item.itemId,
                Office.MailboxEnums.RestVersion.v2_0);
            let rest_url = Office.context.mailbox.restUrl +
                           "/v2.0/me/messages('" +
                           item_rest_id +
                           "')";
            rest_url += "?$expand=SingleValueExtendedProperties($filter=PropertyId eq 'String {00020329-0000-0000-C000-000000000046} Name cecp-<app-guid>')";

            let auth_token = asyncResult.value;
            $.ajax(
                {
                    url: rest_url,
                    dataType: 'json',
                    headers:
                        {
                            "Authorization":"Bearer " + auth_token
                        }
                }
                ).done(
                    function (item) {
                        console.log(JSON.stringify(item));
                    }
                ).fail(
                    function (error) {
                        console.log(JSON.stringify(error));
                    }
                );
        } else {
            console.log(JSON.stringify(asyncResult));
        }
    }
);
```

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[5733c68](https://github.com/OfficeDev/outlook-dev-docs/pull/466/commits/5733c68d6f6743db395ff7a0416c35fa299ee1ce)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[5733c68](https://github.com/OfficeDev/outlook-dev-docs/pull/466/commits/5733c68d6f6743db395ff7a0416c35fa299ee1ce)_: 

:clock10: **Incremental building**: average incremental building time is _17 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[5733c68](https://github.com/OfficeDev/outlook-dev-docs/pull/466/commits/5733c68d6f6743db395ff7a0416c35fa299ee1ce)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/metadata-for-an-outlook-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-add-rest-example/docs/add-ins/metadata-for-an-outlook-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C20%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201812200549255973-466%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=y1oOJ43mQnOm1i1OfZGKMDNtR9byvLAN8%2BuBmMAIE0M%3D&st=2018-12-20T05%3A45%3A30Z&se=2019-01-20T05%3A50%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**